src/lib/mdns/Advertiser.h

```diff
@@ -129,6 +183,9 @@ class ServiceAdvertiser
     /// Advertises the CHIP node as a commisioning node
     virtual CHIP_ERROR Advertise(const CommisioningAdvertisingParameters & params) = 0;
 
+    /// Advertises the CHIP node as a commisionable node
+    virtual CHIP_ERROR Advertise(const CommisionableAdvertisingParameters & params) = 0;
+
```

Comment:
is there a method to stop this advertisement?

Reply:
Not currently - we assume that 'once a chip device, always a chip device and we advertise'. Having a 'StopAdvertising' seems possible though.